src/components/ProfilePage.js

```diff
@@ -42,21 +54,25 @@ export default function ProfilePage(props) {
     emContactName: "name",
     emContactPhone: "123456789"
   }
+  if(currentEmployee === undefined) {
+    currentEmployee = employees[1] //Failsafe for now if the user lands on this page w/o a employee being selected
```

Comment:
Could you please change it so it goes back to the employees table instead of selecting a random employee's details?

Reply:
That's a really good idea, I was trying to get it to work earlier but I think this solution solves the issue. Thank You!